enhancements/assets-generation/cloud-foundry-application-discovery/README.md

```diff
@@ -119,44 +119,42 @@ See [metadata specification](#metadata-specification). |
 | **no-route** | Y | Routes | Processes will have no route information in the canonical form manifest. See [process specification](#process-specification). |
 | **processes** | Y | Processes | See [process specification](#process-specification) |
 | **random-route** | Y | Routes | See [route specification](#route-specification). |
-| **default-route** | Y |  | The default behavior for CF Applications is to define a route based on the name of the application and its domain ([https://docs.cloudfoundry.org/devguide/deploy-apps/deploy-app.html\#default-push](https://docs.cloudfoundry.org/devguide/deploy-apps/deploy-app.html#default-push)).
-Example:
-https://docs.cloudfoundry.org/devguide/multiple-processes.html |
+| **default-route** | Y |  | The [default behavior](https://docs.cloudfoundry.org/devguide/deploy-apps/deploy-app.html#default-push) for CF Applications is to define a route based on the name of the application and its domain. See an [example](https://docs.cloudfoundry.org/devguide/multiple-processes.html). |
```

Comment:
I've removed this field because it doesn't exist in V3

Reply:
rebased, please double check if everything is ok